### PR TITLE
[Symfony 6] Make Sylius installable with Symfony 6

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | symfony-6                 |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets |  replaces #14183, contributes to #13274 |
| License         | MIT                                                          |

The goal of this PR is to make the whole Sylius installable (not working yet!) with Symfony 6 packages. Packages builds are already working after https://github.com/Sylius/Sylius/pull/14287, https://github.com/Sylius/Sylius/pull/14284 and https://github.com/Sylius/Sylius/pull/14285. 

The next steps would be fixing static checks and PHPSpec, Behat tests are waiting for https://github.com/Sylius/Sylius/pull/14283 🖖

### CODE REVIEW

**Review by GSadee — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -181,18 +178,18 @@
...
-        "symfony/password-hasher": "^6.0",
-        "symfony/cache": "^6.0",
-        "symfony/amqp-messenger": "^6.0",
-        "symfony/doctrine-messenger": "^6.0",
-        "symfony/error-handler": "^6.0",
+        "symfony/password-hasher": "<5.4.0",
+        "symfony/cache": "<5.4.0",
+        "symfony/amqp-messenger": "<5.4.0",
+        "symfony/doctrine-messenger": "<5.4.0",
+        "symfony/error-handler": "<5.4.0",
```
Comment (GSadee) [inline composer.json]: Why are you changing versions to be lower than 5.4? 🤔

Comment (Zales0123) [reply composer.json]: I don't know XD I don't know what happened here, fix incoming 🚀

File: composer.json [hunk 2]
```diff
@@ -330,5 +328,8 @@
             "cache:clear": "symfony-cmd",
             "assets:install %PUBLIC_DIR%": "symfony-cmd"
         }
-    }
+    },
+    "repositories": [
+        { "type": "vcs", "url": "git@github.com:Zales0123/SymfonyMockerContainer.git" }
+    ]
```
Comment (Zales0123) [inline composer.json:324]: Btw, this is obviously a temporary solution before we migrate to something else or add support for Symfony 6 on the MockerContainer itself 🖖

File: composer.json (hunk 1 above)
Comment (Zales0123) [reply composer.json]: Oh, I know, these are conflicts 🤦 but I believe we can just remove them 🖖

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Mateusz! :1st_place_medal:

**Pull request merged**